ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py

```diff
@@ -1,6 +1,6 @@
 import ivy
 from ivy.functional.frontends.numpy.func_wrapper import to_ivy_arrays_and_back
-from ivy.func_wrapper import with_unsupported_dtypes
+from ivy.func_wrapper import with_unsupported_dtypes, with_supported_dtypes
```

Comment:
Don't know why you imported the `with_supported_dtypes` here since it is not being referenced to anywhere in the file?